.github/workflows/nightly.yml

```diff
@@ -30,6 +30,8 @@ jobs:
       matrix:
         profiles:
           - '-Pspark-master -pl externals/kyuubi-spark-sql-engine -am'
+        exclude-tags:
+          - '-Dmaven.plugin.scalatest.exclude.tags=org.scalatest.tags.PySparkTest'
```

Comment:
add to pom.xml instead

Reply:
ok, moved to pom.xml and remove this `exclude-tags` from nightly workflow.